src/tests/integration-test/src/queries_test.cpp

```diff
@@ -1476,16 +1474,15 @@ BOOST_AUTO_TEST_CASE(TestDefaultValues) {
   if (!SQL_SUCCEEDED(ret))
     BOOST_FAIL(GetOdbcErrorMessage(SQL_HANDLE_STMT, stmt));
 
-  // Checking that columns are not null.
+  // Checking that the first three columns are not null.
+  // flag, rebuffering_ratio, and video_startup_time will be null.
   for (SQLSMALLINT i = 0; i < 3; ++i)
     BOOST_CHECK_NE(columnLens[i], SQL_NULL_DATA);
 
-  BOOST_CHECK_EQUAL(defaultBoolColumn, false);
-  BOOST_CHECK_EQUAL(defaultDoubleColumn, 0.0);
-  BOOST_CHECK_EQUAL(defaultBigintColumn, 0);
-  BOOST_CHECK_EQUAL(columnLens[3], 1);
-  BOOST_CHECK_EQUAL(columnLens[4], 8);
-  BOOST_CHECK_EQUAL(columnLens[5], 8);
+  BOOST_CHECK_EQUAL(doubleColumn, 63.7);
+  BOOST_CHECK_EQUAL(columnLens[0], 8);
+  BOOST_CHECK_EQUAL(columnLens[1], 16);
+  BOOST_CHECK_EQUAL(columnLens[2], 8);
```

Comment:
These changes are unrelated to the changes to add OLE DB support. The table in Timestream for LiveAnalytics simply has different data than what the test expects.

Reply:
It might be worth generalizing tests to separate the data from the test execution. Then the tests can be applied to different data sets easily. (not in this PR).